tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -94,20 +100,49 @@ void UILabel::InitFromStory(XIBObject* obj) {
         _numberOfLines = atoi(getAttrAndHandle("numberOfLines"));
     }
 
-    if (getAttrib("textAlignment")) {
-        const char* pAlign = getAttrib("textAlignment");
+    const char* lineBreakModeAttributeString = "lineBreakMode";
+    const char* lineBreakModeAttributeValue = getAttrib(lineBreakModeAttributeString);
```

Comment:
because it might be a value that is handled. as you second comment pointed out, i added a warning message in such case.

---
In reply to: [95921835](https://github.com/Microsoft/WinObjC/pull/1701#discussion_r95921835) [](ancestors = 95921835)